### ISSUE TITLE
any plan for the faster R CNN?

### ISSUE THREAD

**Issue by jkh911208** [tflearn/tflearn#756]
I uses lots of convnet code from examples
I am planning on train the model and detect the object on the image
I found that faster-R-CNN is the best way of doing so
but I cannot find any good source
do you have any plan of implement it into example/image so many people can utilize it
thank you

**Comment by willduan** [comment]
I agree

**Comment by jkh911208** [comment]
After I write this issue and did few more research on the object detection with the deep learning network
and i found that Faster R CNN is actually not fast.....
the YOLOv2 and SSD is resulted out much more faster the Faster-R-CNN 
there are still some argument about which one is faster between YOLOv2 and SSD

**Comment by aymericdamien** [comment]
Yes, it would be great to have either Faster RCNN or YOLO implementation example in TFLearn. I will let this thread open with 'contribution welcome', if anyone want to help with that!

**Comment by PanagiotisPtr** [comment]
It looks like I am a bit late to the party but I had the same issue recently.
I found this repository quite helpful: https://github.com/edwardbi/DeepLearningModels/tree/master/RCNN

Maybe it can be helpful to you too. ;)
